src/hooks/useAxios.js

```diff
@@ -0,0 +1,27 @@
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+axios.defaults.baseURL = 'https://api.boardgameatlas.com';
+
+const useAxios = (axiosParams) => {
+  const [response, setResponse] = useState(undefined);
```

Comment:
I would probably recommend using `null` instead of `undefined`. 
```suggestion
  const [response, setResponse] = useState(null);
```